app/models/group.rb

```diff
@@ -28,7 +27,9 @@ def start_date_in_the_past
     if start_date && start_date < Date.today
       errors.add(
         :start_date,
-        I18n.t("activerecord.errors.models.group.start_date_in_the_past_error"),
+        I18n.t(
+          "activerecord.errors.models.group.start_date_in_the_past_error"
+        ),
```

Comment:
Actually, does group still have start and end dates? I thought they should have been removed in favour of Trip?

Reply:
Yes, I will do a proper PR because I have to modify almost all the files related to group.. :)